src/Solr/Solarium/Adapter.php

```diff
@@ -615,6 +615,18 @@ protected function applyParametersOnQuery(
                 $query->setSorts($sortings);
             }
 
+            $queryParser = $parameters->getQueryParser();
+            if ($queryParser === 'edismax') {
+                // get the edismax component
+                $edismax = $query->getEDisMax();
+
+                // NOTE: query is now an edismax query
+                $queryFields = $parameters->getQueryFields();
+                if ($queryFields !== null) {
+                    $edismax->setQueryFields($queryFields);
```

Comment:
Der `queryFields` String enthält die Gewichtungen, richtig? Auf welcher Ebene soll die Auswertung der Konfiguration erfolgen und dieser String konstruiert werden?

Reply:
Was war unsere Entscheidung? Irgendwo dokumentiert?

Reply:
Ich weiß immer noch nicht wo die Gewichtung her kommt. Die Konfiguration taucht nicht in diesem PR auf. Das kann nicht in der Verantwortung der Applikation liegen, auch wenn dort die Default-Konfiguration für Instanzen liegt.

Reply:
Der Setter `setWeightedFields()` akzeptiert einen Array mit Boost Faktoren pro Feld, z.B.: `[ 'title' => 10, 'abstract' => 0.5 ]`. Ist dieser nicht gesetzt, werden alle Felder gleich gewichtet (unabhängig davon welcher Query Parser verwendet wird).

Wir besprechen dies vielleicht am Besten nochmals im nächsten Meeting.